src/core/tests/system.test.ts

```diff
@@ -333,35 +333,103 @@ test('sorts media queries when responsive object values are used', () => {
   ]);
 });
 
-test('supports custom get function', () => {
```

Comment:
Thoughts on if it makes sense to support a custom get also?

Reply:
I can't think of a use case at the moment, but it could offer a decent amount of flexibility

Reply:
I think it's something to keep in mind, but I don't think it's super necessary right now. This could open the possibility of using different prefixes, but I'm not confident that this would be a desireable feature. I'd be interested to hear about how other users of this library would want to use it, but for now, I think it's okay to drop